index.html

```diff
@@ -50,10 +50,200 @@ <h1 class="headline-paragraphs" id="h1">Hey there. I’m Mateo<br><span id="p3">
+          </div>
+          <div class="flex-item-2 flex-container techs-items-font font-family">
+            <p class="techs-items-basic techs-items-1">Ruby on rails</p>
+            <p class="techs-items-basic techs-items-2">css</p>
+            <p class="techs-items-basic techs-items-3">JavaScript</p>
+            <p class="techs-items-basic techs-items-4">html</p>
+          </div>
+          <div class="flex-item-1 flex-container-2">
+            <a href="#" class="bttn font-family">See Project</a>
+          </div>
+        </div>
+      </div> 
+    </div>
+    <div class="grid-container">
+      <div class="grid-item-1">
+      </div>
+      <div class="grid-item-2">
+        <div class="flex-container-2 works-title font-family">
+          <div class="flex-item-1 flex-container-2">
+            <h2 class="card-font card-Title font-family">Multi-Post Stories<br><span>Gain+Glory</span></h2>
+          </div>
+          <div class="flex-item-2 flex-container techs-items-font font-family">
+            <p class="techs-items-basic techs-items-1">Ruby on rails</p>
+            <p class="techs-items-basic techs-items-2">css</p>
+            <p class="techs-items-basic techs-items-3">JavaScript</p>
+            <p class="techs-items-basic techs-items-4">html</p>
+          </div>
+          <div class="flex-item-1 flex-container-2">
+            <a href="#" class="bttn font-family">See Project</a>
+          </div>
+        </div>
+      </div> 
+    </div>
+    <div class="grid-container">
+      <div class="grid-item-1">
+      </div>
+      <div class="grid-item-2">
+        <div class="flex-container-2 works-title font-family">
+          <div class="flex-item-1 flex-container-2">
+            <h2 class="card-font card-Title font-family">Multi-Post Stories<br><span>Gain+Glory</span></h2>
+          </div>
+          <div class="flex-item-2 flex-container techs-items-font font-family">
+            <p class="techs-items-basic techs-items-1">Ruby on rails</p>
+            <p class="techs-items-basic techs-items-2">css</p>
+            <p class="techs-items-basic techs-items-3">JavaScript</p>
+            <p class="techs-items-basic techs-items-4">html</p>
+          </div>
+          <div class="flex-item-1 flex-container-2">
+            <a href="#" class="bttn font-family">See Project</a>
+          </div>
+        </div>
+      </div> 
+    </div>
+  </section>
+  <section class="bg-img" id="aboutme-bg">
+    <div class="flex-container flex-title-content">
+      <h2 class="aboutme-title font-family">About me</h2>
+      <p class="headline-paragraphs p2 font-family"> I can help you build a product, 
+        feature or website Look through some of my work and experience!
+         If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>  
+      <a href="#" class="bttn font-family" id="aboutme-bttn">Get my Resume</a>
+    </div>
```

Comment:
- [ ] semantic tags make it clear to the browser what the meaning of a page and its content is. Let us be familiar of using semantic code, kindly, change div to article

Reply:
Done :heavy_check_mark:

Reply:
Well resolved